sense-android-library/documentation/sensor_profiles.md

```diff
@@ -0,0 +1,472 @@
+      "type": "integer"
+    },
+    "question_id": {
+      "description": "The identifier of the question (e.g. stylemsg_id)",
+      "type": "integer"
+    },
+    "answer_id": {
+      "description": "The identifier of the answer (e.g. 2, for a 10 point scale 0-9)",
+      "type": "integer"
+    },
+    "answer": {
+      "description": "The string representation of the answer (e.g. 30%)",
+      "type": "string"
+    },
+    "task_type": {
+      "description": "The Task.TYPE representing the domain to which this question belongs to (e.g. MR_TASK_DOMAIN_1)",
+      "type": "string"
+    }
+  },
+  "required": [
+    "question",
+    "score",
+    "question_id",
+    "answer_id",
+    "answer",
+    "task_type"
+  ]
+}
+~~~
+
+### time_zone
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The time-zone id, e.g. Europe/Paris",
+      "type": "string"
+    },
+    "offset": {
+      "description": "The offset from GMT in seconds",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "offset"
+  ]
+}
+~~~
+
+## Sensing Library sensors
+These sensors contain data which is stored in the back-end to improve the current cortex modules.
+
+### accelerometer
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "x-axis": {
+      "description": "The acceleration force applied on the x-axis in m/s2",
+      "type": "number"
+    },
+    "y-axis": {
+      "description": "The acceleration force applied on the y-axis in m/s2",
+      "type": "number"
+    },
+    "z-axis": {
+      "description": "The acceleration force applied on the z-axis in m/s2",
+      "type": "number"
+    }
+  },
+  "required": [
+    "x-axis",
+    "y-axis",
+    "z-axis"
+  ]
+}
+~~~
+
+### battery
+*renamed from battery_sensor*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "status": {
+      "description": "The status of the battery, e.g. charging, discharging, full",
+      "type": "string"
+    },
+    "level": {
+      "description": "The battery level in percentage.",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "level"
+  ]
+}
+~~~
+### call
+*renamed from call_state*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "state": {
+      "description": "The state of the phone",
+      "enum": ["idle", "dialing", "ringing", "calling"]
+    },
+    "incomingNumber": {
+      "description": "The phone number of the in comming call",
+      "type": "string"
+    },
+    "outgoingNumber": {
+      "description": "The phone number of the out going call",
+      "type": "string"
+    }
+  },
+  "required": [
+    "state"
+  ]
+}
+~~~
+### light
+
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The illuminance in lx",
```

Comment:
@tedschmidt i know that lux can be abbreviate as well as lx, but i guess it'll be more readable in full.